pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/painless/UpsertPainless.java

```diff
@@ -46,7 +46,14 @@ public void export() {
                     "created/updated %s records in ES data for %s", getNumberOfUpserted(bulkByScrollResponse), generator.getPropertyName())
             );
         } catch (IOException e) {
-            throw new RuntimeException("Error occurred while exporting data to ES", e);
+            try {
+                BulkByScrollResponse bulkByScrollResponse = clientInstance.updateByQuery(updateByQueryRequest, RequestOptions.DEFAULT);
+                logger.info(String.format("created/updated %s records in ES data for %s", getNumberOfUpserted(bulkByScrollResponse),
+                        generator.getPropertyName())
+                );
+            } catch (IOException retryE) {
+                throw new RuntimeException("Error occurred while exporting data to ES", retryE);
+            }
```

Comment:
I think retry on exception is not a good solution for the error mentioned in the description, there are doubts why it happens:
https://stackoverflow.com/questions/56840637/version-conflict-when-using-the-delete-method-of-elasticsearch-dsl
https://www.elastic.co/guide/en/elasticsearch/guide/master/optimistic-concurrency-control.html

maybe joining those two insertion `insertGenomicIdForParticipant() && insertExitStatusForParticipant()` could fix the problem, if I got correctly from the links why it happens.